todos-app/backend/task_service/spec/requests/task_spec.rb

```diff
@@ -117,6 +117,28 @@
           expect(response.status).to eq(400)
         end
       end
+
+      context "with valid search param" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.